src/commissaire_service/clusterexec/__init__.py

```diff
@@ -43,6 +44,7 @@ def __init__(self, exchange_name, connection_url):
             {'routing_key': 'jobs.clusterexec.*'}
         ]
         super().__init__(exchange_name, connection_url, queue_kwargs)
+        self.storage = StorageClient(self)
```

Comment:
Makes sense. An idea for the future: How about a ``CommissaireService`` subclass or mixin that can be used by other classes?